source/editableText.py

```diff
@@ -259,6 +259,12 @@ class EditableTextWithoutAutoSelectDetection(EditableText):
 	This should be used when an object does not notify of selection changes.
 	"""
 
+	def waitForAndSpeakSelectionChange(self, oldTextInfo):

```

Comment:
nit: This should perhaps be called `waitForAndReportSelectionChange`, as it affects braille as well, not just speech.